src/views/AccountingOverview/internal/TransactionsToReview.tsx

```diff
@@ -65,12 +67,22 @@ export function TransactionsToReview({
     }
   }
 
+  const labelDataProperties = useMemo(() => toDataProperties({ size }), [size])
+  let verticalGap: StackProps['gap'] = '3xs'
+  switch (size) {
+    case 'sm':
+      verticalGap = '3xs'
+      break
+    case 'lg':
+      verticalGap = 'sm'
+      break
+  }
   return (
     <div onClick={onClick} className={CLASS_NAME}>
-      <VStack gap='3xs' align='start'>
-        <Text size={size === 'lg' ? TextSize.lg : TextSize.sm}>
+      <VStack gap={verticalGap} align='start'>
+        <h3 className={HEADING_CLASS_NAME} {...labelDataProperties}>
           Transactions to review
-        </Text>
+        </h3>
```

Comment:
Suggestion - make this `H3` its own component.

Reply:
done